### PR TITLE
team_id in refund

### PR DESCRIPTION
**Pull request by nim-odoo**
Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by wtaferner** [comment]
@nim-odoo 
It was eventually removed by @tivisse while forward porting...you could take my original PR https://github.com/odoo/odoo/pull/17065 as team_id should be in ```_get_refund_common_fields```

**Comment by tivisse** [comment]
@wtaferner It has been removed because I merged it in master it isn't present in this branch?

Furthermore  ´team_id´ is a field from sales, so not compatible with account.

**Comment by wtaferner** [comment]
@tivisse 
Thx, @mart-e told me a second a ago :-)

File: addons/sale/models/account_invoice.py [hunk 1]
```diff
@@ -89,6 +89,9 @@ def get_delivery_partner_id(self):
         self.ensure_one()
         return self.partner_shipping_id.id or super(AccountInvoice, self).get_delivery_partner_id()
 
+    def _get_refund_common_fields(self):
+        super(AccountInvoice, self)._get_refund_common_fields() + ['team_id']
```
Comment (wtaferner) [inline addons/sale/models/account_invoice.py]: return

**Comment by wtaferner** [comment]
What about the ```shipping_partner_id``` we talked about?

**Pull request merged**